crt_portal/static/js/word_count.js

```diff
@@ -76,27 +82,31 @@ function updateWordCount (e, max, textAreaElem) {
     onBelowLimit(wordCount, max, textAreaElem);
   }
 
-  if (wordCount >= (max * 4 / 5)) {
+  if (wordCount >= (max * 4) / 5) {
     wordCountArea.setAttribute('aria-live', 'assertive');
   } else {
     wordCountArea.setAttribute('aria-live', 'polite');
   }
-};
+}
 
-function listenWordCount (e) {
+function listenWordCount(e) {
   if (textAreaElem500.length > 0) {
-    updateWordCount(e, max=500, textAreaElem=document.getElementById(textAreaElem500[0].id));
-  };
+    updateWordCount(
+      e,
+      (max = 500),
+      (textAreaElem = document.getElementById(textAreaElem500[0].id))
+    );
+  }
 
   if (textAreaElem10.length > 0) {
-    updateWordCount(e, max=10, textAreaElem=document.getElementById(textAreaElem10[0].id));
-  };
-};
+    updateWordCount(e, (max = 10), (textAreaElem = document.getElementById(textAreaElem10[0].id)));
+  }
```

Comment:
I am noticing it is removing some of the semicolons, I thought those were needed for older versions of IE. I could be wrong on that.

Reply:
In this case it's fixing a real JS error; there shouldn't be any semicolons following `if (a) { b }` statements in JavaScript. 

See "Avoid! ... After a closing curly bracket" in this article: [Your Guide to Semicolons in JavaScript
](https://news.codecademy.com/your-guide-to-semicolons-in-javascript/).